### PR TITLE
Changelog 1.7.8.3

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Add changelog for 1.7.8.3
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27217)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by sowbiba — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -23,6 +23,54 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 
 Release Notes for PrestaShop 1.7
 
+####################################
+#   v1.7.8.3 - (2022-01-XX)
```
Comment (atomiix) [inline docs/CHANGELOG.txt]:
```suggestion
#   v1.7.8.3 - (2022-01-05)
```

**Review by atomiix — DISMISSED** [review]

File: docs/CHANGELOG.txt (hunk 1 above)
Comment (PululuK) [reply docs/CHANGELOG.txt]: @atomiix You're sure ?
image
https://github.com/PrestaShop/PrestaShop/issues/26969

Comment (atomiix) [reply docs/CHANGELOG.txt]: This date is the build date, not the release date! But I'm still wrong, it was built on the 5th 😅 I'll update my suggestion

Comment (PierreRambaud) [reply docs/CHANGELOG.txt]: I agree with @atomiix

**Review by PierreRambaud — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Pull request merged**